### ISSUE TITLE
9.25.3 Nginx (X-Accel-Redirect) not working

### ISSUE THREAD

**Issue by shoeper** [owncloud/core#14160]
### Steps to reproduce
1. Download file -> works
2. Enable 9.25.3 Nginx (X-Accel-Redirect)
3. Download file -> doesn't work

### Expected behaviour
When following http://doc.owncloud.org/server/8.0/ownCloudAdminManual.pdf -> Configuration -> Performance tips -> 9.25 Serving Static Files for Better Performance -> 9.25.3 Nginx (X-Accel-Redirect) downloading files should work

### Actual behaviour
I get the error `ERR_RESPONSE_HEADERS_MULTIPLE_CONTENT_DISPOSITION`in chrome

### Server configuration
**Operating system:** Debian (Wheezy) 3.2.65-1+deb7u1 x86_64

**Web server:** Nginx 1.2.1

**Database:** 5.5.41-0+wheezy1

**PHP version:** PHP 5.4.36-0+deb7u3

**ownCloud version:** ownCloud 8.0 (stable)

**Updated from an older ownCloud or fresh install:** no

**The content of nginx config:**

```
...
        location ~ \.php(?:$|/) {
                fastcgi_split_path_info ^(.+\.php)(/.+)$;
                include fastcgi_params;
                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
                fastcgi_param PATH_INFO $fastcgi_path_info;
                fastcgi_param HTTPS on;
                fastcgi_param MOD_X_ACCEL_REDIRECT_ENABLED on;
                fastcgi_pass php-handler;
        }

        location ^~ /data {
               internal;
        }
...
```

**Are you using external storage, if yes which one:** local

**Are you using encryption:** no

### Client configuration
**Browser:** Chrome

**Operating system:** Windows 8.1

#### ownCloud log (data/owncloud.log)
```
Cannot modify header information - headers already sent by (output started at /var/www/owncloud/lib/private/image.php:249) at /var/www/owncloud/lib/private/appframework/app.php#104
```

**Comment by RealRancor** [comment]
Not sure but this could be related to: https://github.com/owncloud/core/pull/14090

**Comment by josh4trunks** [comment]
@RealRancor we're presenting 2 different solutions to the same issue.

**Comment by josh4trunks** [comment]
Like @RealRancor said, this was changed in OC8. Please refer to OC6 docs if you want to get X-Accel working right now. Hopefully well fix this soon.

**Comment by PVince81** [comment]
@josh4trunks was there any updated related to that ?

**Comment by josh4trunks** [comment]
this will be fixed by https://github.com/owncloud/core/pull/14125

I'll test and document the fix tonight.

**Comment by josh4trunks** [comment]
this was fixed by https://github.com/owncloud/core/pull/14125
it should be available in 8.0.3
